.github/workflows/zowe-exlorer-ci.yml

```diff
@@ -0,0 +1,48 @@
+name: Zowe Explorer CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      # Continue to run tests on the other systems if one fails
+      fail-fast: false
+      matrix:
+        node-version: [10.x, 12.x]
+        # order operating systems from best to worst
+        os: [windows-latest, ubuntu-latest, macos-latest]
+
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - run: npm install
+
+    - name: Audit from Windows / Node 12.x
+      run: npm audit --audit-level=moderate
+      if: matrix.os == 'windows-latest' && matrix.node-version == '12.x'
+
+    - name: Create test data file
+      run: cp resources/testProfileData.example.ts resources/testProfileData.ts
+
+    - run: npm run build --if-present
+
+    - run: npm test
+      env:
+        CI: true
+        NODE_OPTIONS: --max_old_space_size=4096
+
+    - name: Upload test results from Windows / Node 12.x
```

Comment:
Is it test results or aritfacts? Kind of confused by the two terms in here?

Reply:
`upload-artifact@v1` is the package name for storing artifacts.  The artifact I was uploading is the test results folder.  If you have a preference for wording, please let me know, and I'll adjust it.

Reply:
I think it is just me not being familiar with github actions and the fact that the upload is only done for windows / node 12.x . I would expect us to upload everything so that when something fails, we could look at the log regardless of the platform and node version.

Reply:
i changed this to upload for all combinations